### ISSUE TITLE
CLI - db:version - colors not defined

### ISSUE THREAD

**Issue by combsco** [keithwhor/nodal#131]
> Nodal 0.5.7

**nodal db:version**
```
SELECT ("schema_migrations"."id") AS "id" FROM schema_migrations AS "schema_migrations" ORDER BY "schema_migrations"."id" desc LIMIT 1
[]
21ms

/Users/crisopolis/npm-global/lib/node_modules/nodal/core/my/bootstrapper.js:110
            console.log(colors.green.bold('Current Schema Version: ') + result.rows[0].id);
                        ^

ReferenceError: colors is not defined
    at PostgresQuery.<anonymous> (/Users/crisopolis/npm-global/lib/node_modules/nodal/core/my/bootstrapper.js:110:25)
    at PostgresQuery.callback (/Users/crisopolis/npm-global/lib/node_modules/nodal/core/required/db/database.js:102:18)
    at PostgresQuery.<anonymous> (/Users/crisopolis/npm-global/lib/node_modules/nodal/node_modules/any-db-postgres/index.js:59:28)
    at emitNone (events.js:85:20)
    at PostgresQuery.emit (events.js:179:7)
    at endReadableNT (_stream_readable.js:906:12)
    at nextTickCallbackWith2Args (node.js:474:9)
    at process._tickDomainCallback (node.js:429:17)
```

**Comment by intabulas** [comment]
Good catch.. Will fix this now

**Comment by intabulas** [comment]
@keithwhor Fixed in #132
